.github/workflows/g_m.yml

```diff
@@ -0,0 +1,14 @@
+name: Docker Image CI
+
```

Comment:
please drop this two patchset (one that adds this file and the next that drops)

Reply:
I don't know how to do it.

Reply:
Does this help? https://asciinema.org/a/eJWdZPqPme3xsqvy5NKXKekO5